Managers/LuaMan.cpp

```diff
@@ -2116,10 +2116,10 @@ int LuaMan::Create()
 
         // NOT a member function, so adopting _1 instead of the _2 for the first param, since there's no "this" pointer!!
         def("DeleteEntity", &DeleteEntity, adopt(_1)),
-        def("PosRand", &PosRand),
+		def("RangeRand", (double(*)(double, double))& RandomNum),
```

Comment:
Can you put a TODO (or an issue) in somewhere to overwrite math.random with our new random functions? I don't wanna lose track of that.
Alternatively, if you wanna try to do it yourself you're totally welcome to.

Reply:
Sure, I'll put in an issue for now.
Issue: #179